crypto/src/test/java/org/springframework/security/crypto/bcrypt/BCryptPasswordEncoderTests.java

```diff
@@ -33,6 +33,14 @@ public void matches() {
 		String result = encoder.encode("password");
```

Comment:
Generally with these tests please split up these tests so that there is a different test for each method